application/squeeze2cast/castcore.c

```diff
@@ -169,7 +169,6 @@ json_t *GetTimedEvent(struct sCastCtx *Ctx, uint32_t msWait) {
 bool LaunchReceiver(tCastCtx *Ctx) {
 	// try to reconnect if SSL connection is lost
 	if (!CastConnect(Ctx)) {
-		pthread_mutex_unlock(&Ctx->Mutex);
```

Comment:
What is the case where we return from CastConnect with mutex locked? It does not seem that it can be the case